cli/testdata/server-config.yaml.golden

```diff
@@ -493,13 +493,15 @@ userQuietHoursSchedule:
 # compatibility reasons, this will be removed in a future release.
 # (default: false, type: bool)
 allowWorkspaceRenames: false
+# Configure how notifications are processed and delivered.
 notifications:
   # Which delivery method to use (available options: 'smtp', 'webhook').
   # (default: smtp, type: string)
   method: smtp
   # How long to wait while a notification is being sent before giving up.
   # (default: 1m0s, type: duration)
-  dispatch-timeout: 1m0s
+  dispatchTimeout: 1m0s
```

Comment:
Looks like we're mixing _camelCase_ and _snake_case_ in the config. Should we stick with one format?

Reply:
Whoops, you're right. Thanks 👍 
We have a general problem with this it seems, but I'll at least try to make all of my additions internally consistent.